scanpy/tests/test_plotting.py

```diff
@@ -1215,6 +1215,76 @@ def test_scatter_specify_layer_and_raw():
         sc.pl.umap(pbmc, color="HES4", use_raw=True, layer="layer")
 
 
+def test_scatter_no_basis_per_obs(image_comparer):
+    """Test scatterplot of per-obs points with no basis"""
+    save_and_compare_images = image_comparer(ROOT, FIGS, tol=15)
+    pbmc = sc.datasets.pbmc68k_reduced()
+    sc.pl.scatter(pbmc, x="HES4", y="percent_mito", color="n_genes", use_raw=False)
+    save_and_compare_images("scatter_HES_percent_mito_n_genes")
+
+
+def test_scatter_no_basis_per_var(image_comparer):
+    """Test scatterplot of per-var points with no basis"""
+    save_and_compare_images = image_comparer(ROOT, FIGS, tol=15)
+    pbmc = sc.datasets.pbmc68k_reduced()
+    sc.pl.scatter(pbmc, x="AAAGCCTGGCTAAC-1", y="AAATTCGATGCACA-1", use_raw=False)
+    save_and_compare_images("scatter_AAAGCCTGGCTAAC-1_vs_AAATTCGATGCACA-1")
+
+
+@pytest.fixture
+def pbmc_filtered():
+    pbmc = sc.datasets.pbmc68k_reduced()
+    sc.pp.filter_genes(pbmc, min_cells=10)
+    return pbmc
+
+
+def test_scatter_no_basis_raw(check_same_image, pbmc_filtered):
+    """Test scatterplots of raw layer with no basis."""
+    path1 = FIGS / "scatter_EGFL7_F12_FAM185A_rawNone.png"
+    path2 = FIGS / "scatter_EGFL7_F12_FAM185A_rawTrue.png"
+    path3 = FIGS / "scatter_EGFL7_F12_FAM185A_rawToAdata.png"
```

Comment:
```suggestion
def test_scatter_no_basis_raw(check_same_image, pbmc_filtered, tmpdir):
    """Test scatterplots of raw layer with no basis."""
    path1 = tmpdir / "scatter_EGFL7_F12_FAM185A_rawNone.png"
    path2 = tmpdir / "scatter_EGFL7_F12_FAM185A_rawTrue.png"
    path3 = tmpdir / "scatter_EGFL7_F12_FAM185A_rawToAdata.png"
```

Could do `tmpdir` instead of `FIGS`? Why `FIGS`?